rdf4j/src/test/java/org/apache/commons/rdf/rdf4j/Rdf4JRDFTermFactoryTest.java

```diff
@@ -18,19 +18,19 @@
 package org.apache.commons.rdf.rdf4j;
 
 import org.apache.commons.rdf.api.AbstractRDFTermFactoryTest;
-import org.apache.commons.rdf.api.RDFTermFactory;
+import org.apache.commons.rdf.api.RDF;
 import org.junit.Assume;
 
 public class Rdf4JRDFTermFactoryTest extends AbstractRDFTermFactoryTest {
 
 	@Override
-	public RDFTermFactory createFactory() {
-		return new RDF4JFactory();
+	public RDF createFactory() {
+		return new RDF4J();
 	}
 	
 	@Override
 	public void testInvalidLiteralLang() throws Exception {
-		Assume.assumeTrue("Sesame doesn't check Lang strings",false);
+		Assume.assumeTrue("RDD4J doesn't check Lang strings",false);
```

Comment:
Fixed. Also renamed `*RDFTermFactoryTest` to `*RDFTest`

I changed to `@Ignore` in `JsonLdRdfTest`, but I can't use @Ignore here as the parent test method has `@Test(expected = IllegalArgumentException.class)`  and somehow the `@Ignore` is not honoured - at least in Eclipse.

Using `Assume.assumeTrue` is still recognized as "test skipped".